### DISCUSSION TITLE
Hosting on decentralized storage?

### DISCUSSION THREAD

**Discussion by nemanjakrstonic** [jekyll/jekyll#9453] [Q&A]
Hi, I’m looking to host my website on decentralized storage. So far, I found this service https://peerspace.network/ that offers hosting and it's similar to Netlify but on Arweave. 

Has anybody tried to migrate to decentralized storage? What was your experience?

**Comment by y377** [comment]
The biggest purpose of decentralization is to get rid of geographical restrictions, but web websites do have language restrictions. For example, if I write a Chinese article, even if I save it, can native African speakers understand my article?
